web/src/locales/en/data.json

```diff
@@ -0,0 +1,809 @@
+    "It is recommended that you do not use real person photos as avatars, even photos of yourself. The use of other people’s photos is prohibited": "It is recommended that you do not use real person photos as avatars, even photos of yourself. The use of other people’s photos is prohibited",
+    "Phone not verified": "Phone not verified",
+    "Email not verified": "Email not verified",
+    "Modify Phone": "Modify Phone",
+    "Phone Verification": "Phone Verification",
+    "QQ Verification": "QQ Verification",
+    "Link with QQAccount": "Link with QQAccount",
+    "Profile": "Profile",
+    "Avatar": "Avatar",
+    "Forum": "Forum",
+    "Email Reminder": "Email Reminder",
+    "Open": "Open",
+    "Close": "Close",
+    "Description on email reminder": "Description on email reminder",
+    "Will send you a email when you receive a new reply": "Will send you a email when you receive a new reply",
+    "Change forum setting successfully": "Change forum setting successfully",
+    "Set Username": "Set Username",
+    "Welcome to": "Welcome to",
+    ", you just registered your": ", you just registered your",
+    "account. Now please set a username here, you can only use half-width English letters and numbers. Other users can send you a message through @ your account name. The user name cannot be changed after setting.": "account. Now please set a username here, you can only use half-width English letters and numbers. Other users can send you a message through @ your account name. The user name cannot be changed after setting.",
+    "save": "save",
+    "Set username failed": "Set user name failed",
+    "Set username success": "Set user name success",
+    "Username existed or email existed": "User name or email existed",
+    "Unbind Account": "Unbind Account",
+    "WeChat Verification": "WeChat Verification"
+  },
+  "notification": {
+    "Total reminders received": "Total reminders received",
+    "Reminder system": "Reminder system",
+    "Thanks for your reply in topic": "Thanks for your reply in topic",
+    "里的回复": "",
+    "Favorite you topic": "Favorite you topic",
+    "Thanks for you topic": "Thanks for you topic",
+    "Replied to you in": "Replied to you in",
+    "Mentioned you in topic": "Mentioned you in topic",
+    "Mentioned you in": "Mentioned you in",
+    "里回复了你": "",
+    "里提到了你": "",
+    "Delete": "Delete"
+  },
+  "forgot": {
+    "Reset password by phone": "Reset password by phone",
+    "Reset password by email": "Reset password by email",
+    "Username": "Username",
+    "Reset Password": "Reset Password",
+    "You can reset the password at most 2 times within 24 hours.": "You can reset the password at most 2 times within 24 hours.",
+    "Continue": "Continue",
+    "Mail": "Mail",
+    "Reset password for account": "Reset password for account",
+    "New password": "New password",
+    "Password again": "Password again",
+    "重新设置密码": "",
```

Comment:
why Chinese here?

Reply:
I think there are Chinese translations wrongly putted in the original repo:

https://github.com/casbin/casnode/blob/master/web/src/locales/en.json#L613

Reply:
I will fix it